.github/workflows/test.yml

```
@@ -45,3 +45,16 @@ jobs:
       - name: Build and run tests
         run: |
           cargo test --verbose --workspace
+
+  fuzz:
+    needs: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dtolnay/rust-toolchain@nightly
+      - name: Checkout
+        uses: actions/checkout@v4
```

It would be nice if the fuzz corpus was cached, so fuzzing for 1 minute is more effective

```suggestion
        uses: actions/checkout@v4
        
      - name: Cache fuzz corpus
        uses: actions/cache@v4
        with:
          key: ${{ runner.os }}-${{ hashFiles('Cargo.lock', 'crates/ansi-to-html/fuzz/fuzz_targets/**') }}
          path: crates/ansi-to-html/fuzz/corpus
```

---

Unfortunately this won't keep re-saving the corpus after the initial cache gets created, but that's still better than always starting with a fresh corpus

I tried looking for some toggle to allow always re-saving a cache to the same key, but didn't see anything obvious